src/BPEssentials/ExtensionMethods/PlayerCooldowns.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BPEssentials.ExtendedPlayer;
+using BrokeProtocol.Entities;
+using UnityEngine;
+
+namespace BPEssentials.ExtensionMethods.Cooldowns
+{
+    public static class ExtensionPlayerCooldowns
+    {
+        public static bool IsCooldown(this SvPlayer player, string type, string key)
```

Comment:
Maybe change to `HasCooldown` or `InCooldown`?
```suggestion
        public static bool HasCooldown(this SvPlayer player, string type, string key)
```
```suggestion
        public static bool InCooldown(this SvPlayer player, string type, string key)
```